adflow/pyADflow.py

```diff
@@ -3128,6 +3149,10 @@ def setAeroProblem(self, aeroProblem, releaseAdjointMemory=True):
             if releaseAdjointMemory:
                 self.releaseAdjointMemory()
 
+        if not self.getOption("ANKCFLReset"):
+            # also set the ANK CFL to the last value if we are restarting
+            self.adflow.anksolver.ank_cfl = aeroProblem.adflowData.ank_cfl
```

Comment:
I assume this just takes the initial default value in the fortran layer on the first iteration when `ANKCFLReset: False`?
(It looks like it from the added id statement in the fortran layer for `ANK_CFL = ANK_CFL0` but I want to double check)

Reply:
Yeah my phrasing was odd but that is what I meant, sounds good